qa-tests-backend/src/test/groovy/ImageSignatureVerificationTest.groovy

```diff
@@ -97,6 +107,22 @@ w9e2Azq1OYIh/pbeBMHARDrBaqqmuMR9+BfAaPAYdkNTU6f58M2zBbuL0A==
             .addLabel("app", "image-without-signature")
             .setNamespace(SIGNATURE_TESTING_NAMESPACE)
 
+    // Deployment holding an image with the same digest as docker.io/daha97/alt-nginx that does not have a cosign
+    // signature associated with it.
+    static final private Deployment SAME_DIGEST_NO_SIGNATURE = new Deployment()
+            .setName("same-digest-without-signature")
+            .setImage("docker.io/daha97/nginx:1.23")
+            .addLabel("app", "image-same-digest-without-signature")
+            .setNamespace(SIGNATURE_TESTING_NAMESPACE)
+
+    // Deployment holding an image with the same digest as docker.io/daha97/nginx that does have a cosign signature
+    // associated with it.
+    static final private Deployment SAME_DIGEST_WITH_SIGNATURE = new Deployment()
+            .setName("same-digest-with-signature")
+            .setImage("docker.io/daha97/alt-nginx:1.23")
+            .addLabel("app", "image-same-digest-with-signature")
+            .setNamespace(SIGNATURE_TESTING_NAMESPACE)
```

Comment:
Will replace those soon™ with images stored within quay.io (now that quay _does_ support uploading & storing cosign signatures).

Reply:
We shouldn't merge in the current state, right?

Reply:
Yea, I switched them now to quay.